In [1]:
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
%matplotlib inline

In [5]:
ds1 = pd.read_csv('Customer Bill Detail.csv')
#ds['Dish Details'].value_counts()
ds1[8373:8375]

Type        Date   Bill Number  Discount  Subtotal  \
8373  Home Delivery  2018-09-22  200837723936       0.0     169.0   
8374  Home Delivery  2018-09-22  200837724264      77.4      51.6   

      Delivery Amt  Total Bill        Channel   Customer ID  \
8373             0         169  Zomato Online  245831671237   
8374             0          52  Zomato Online  138626187589   

                                          Dish Details  
8373  Hyderabadi Chicken Fry Piece Biryani (QTY-1.000)  
8374                         Chicken Tikka (QTY-1.000)

### In row 8374 as shown above, the Discount has exceeded the subtotal which is not appropriate and that is out of place in data. The discount is 77.4 and the subtotal is just 52 so it is not appropriate data

In [8]:
ds3 = pd.read_csv('Market Pricing.csv')
dp=ds3[['Restaurant','Menu Item']][1904:2085]
dp

Restaurant                                  Menu Item
1904       Goli                         Chicken Curry Full
1905       Goli                         Chicken curry Half
1906       Goli                        Butter Chicken Half
1907       Goli                        Butter Chicken Full
1908       Goli                      Tandoori Chicken Half
1909       Goli                      Tandoori Chicken Full
1910       Goli  Hyderabadi Chicken Fry Piece Biryani Half
1911       Goli  Hyderabadi Chicken Fry Piece Biryani Full
1912       Goli         Hyderabadi Mutton Dum Biryani Full
1913       Goli         Hyderabadi Mutton Dum Biryani Half
1914       Goli                           Rajma Rice Combo
1915       Goli                           Chole Rice Combo
1916       Goli                    Lemon Rice Combo [Full]
1917       Goli                               Veg Platter 
1918       Goli                      Non Veg Royal Platter
1919       Goli                            Chicken Platter
1920       Goli                                    Omlatte
1921       Goli                              Bread Omlatte
1922       Goli                              Chole Bathure
1923       Goli                     Chole Kulche Amritsari
1924       Goli                  Chicken Garlic Tikka Half
1925       Goli                  Chicken Garlic Tikka Full
1926       Goli                  Chicken Pudina Tikka Full
1927       Goli                  Chicken Pudina Tikka Half
1928       Goli                        Paneer Garlic Tikka
1929       Goli                        Paneer Pahadi Tikka
1930       Goli                        Paneer Pudina Tikka
1931       Goli                      Chicken Roast Special
1932       Goli               Chicken Leg Malai [2 Pieces]
1933       Goli              Chicken Leg Afgani [2 Pieces]
...         ...                                        ...
2055       Goli                                Matar Pulav
2056       Goli                              Tadka Khichdi
2057       Goli                               Veg Biryani 
2058       Goli                                Egg Biryani
2059       Goli           Special Hyderabadi Keema Biryani
2060       Goli  Hyderabadi Chicken Fry Piece Biryani Half
2061       Goli  Hyderabadi Chicken Fry Piece Biryani Full
2062       Goli         Hyderabadi Mutton Dum Biryani Full
2063       Goli         Hyderabadi Mutton Dum Biryani Half
2064       Goli                                  Veg Roll 
2065       Goli                                  Egg Roll 
2066       Goli                                Paneer Roll
2067       Goli                               Chicken Roll
2068       Goli                        Chilly Chicken Full
2069       Goli                        Chilly Chicken Half
2070       Goli                         Chilly Paneer Full
2071       Goli                         Chilly Paneer Half
2072       Goli                             Veg Manchurian
2073       Goli                         Chicken Manchurian
2074       Goli                Chicken Lollipop [6 Pieces]
2075       Goli                                 Fried Rice
2076       Goli                             Egg Fried Rice
2077       Goli                         Chicken Fried Rice
2078       Goli                                Green Salad
2079       Goli                            Kachumber Salad
2080       Goli                              Roasted Papad
2081       Goli                                  Fry Papad
2082       Goli                               Masala Papad
2083       Goli                              Panjabi Salad
2084       Goli                                Onion Salad

[181 rows x 2 columns]

In [9]:
dp['Menu Item'].value_counts()

Butter Chicken Half                          2
Hyderabadi Chicken Fry Piece Biryani Full    2
Chicken Malai Tikka Full                     2
Tandoori Chicken Full                        2
Hyderabadi Mutton Dum Biryani Half           2
Hyderabadi Mutton Dum Biryani Full           2
Chicken Curry Full                           2
Butter Chicken Full                          2
Chicken Pahadi Tikka Full                    2
Chicken Pahadi Tikka Half                    2
Tandoori Chicken Half                        2
Chicken Malai Tikka Half                     2
Hyderabadi Chicken Fry Piece Biryani Half    2
Chole Rice Combo                             1
Tandoori Chicken Kali Mirch Full             1
Chicken Home Style Half                      1
Cheese Garlic Paratha                        1
Dhaniya Chicken Half                         1
Egg Curry                                    1
Paneer Garlic Tikka                          1
Chicken Amritsari Tikka Half                 1
Chilly Chicke

### In goli's Menu some dishes are listed twice which is also not appropriate